docs/source/generate_agent_list.py

```diff
@@ -94,9 +46,10 @@ def write_all_agents():
         )
     ]
     for agent in agents:
-        with open(f'agent_refs/{agent}.rst', 'w') as fout:
-            write_agent(fout, prepare_agent_readme(agent))
+        with open(f'agent_refs/{agent}.md', 'w') as fout:
+            fout.write(prepare_agent_readme(agent))
```

Comment:
can you please include a screenshot of what the new agents file looks like? this didn't render well for me at all but i am assuming with the new style it does?

Reply:
<img width="1552" alt="Screen Shot 2019-12-30 at 10 44 46 AM" src="https://user-images.githubusercontent.com/31896/71589026-78c6c980-2af1-11ea-8c16-d18459c306c9.png">